app-engine/src/main/java/be/cytomine/appengine/handlers/StorageStringEntry.java

```diff
@@ -0,0 +1,20 @@
+package be.cytomine.appengine.handlers;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@NoArgsConstructor
+public class StorageStringEntry extends StorageDataEntry {
+
+    private String dataAsString;
+
+    public StorageStringEntry(String data, String name, StorageDataType storageDataType) {
+        this.dataAsString = data;
+        super.setName(name);
+        super.setStorageDataType(storageDataType);
+    }
```

Comment:
So now, whit that inheritance, If I have a `List<StorageDataEntry>`, I have no idea, by default if the data is in `data` or `dataAsString`. 
Could you maybe use some genericity here? 
`Storage<T>` 

(I don't like that either actually, but I don't know your use case)
Do you really need to pack those two (StorageDataEntry, StorageStringEntry) in the same place at the end?